modules/sugar/src/main/ts/ephox/sugar/api/properties/Css.ts

```diff
@@ -83,11 +85,11 @@ const get = function (element: Element<DomElement>, property: string): string |
   // Turns out we do this a lot.
   const v = (r === '' && !Body.inBody(element)) ? getUnsafeProperty(dom, property) : r;
 
-  // undefined is the more appropriate value for JS. JQuery coerces to an empty string, but screw that!
-  return v === null ? undefined : v;
+  // Older browsers may return null instead of an empty string, so normalize back to an empty string
```

Comment:
This is probably the largest change. There's very, very few places we were checking for `undefined` in `Css.get` plus it goes against the normal browser behaviour which is to return an empty string. Additionally I'm not sure how this would even return null as the types don't say it can, so I figured this was better than adding heaps of is undefined checks.

Reply:
Maybe we should notify QA of this, in case they see anything weird with e.g. UI detecting styles incorrectly, dialogs doing weird things around CSS values, etc.?

Reply:
Yeah either of those are possible. I had also wondered if once upon a time it used https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/getPropertyCSSValue which can return null.

Reply:
no, at worst it would've used `elem.style[property]`

- [ ] remove null check and empty string fallback

Reply:
Done